$MirOS: ports/net/tor/patches/patch-configure_in,v 1.2 2007/08/06 09:47:34 tg Exp $
--- configure.in.orig	Thu Jan 17 19:49:03 2008
+++ configure.in	Tue Mar  4 02:15:39 2008
@@ -44,7 +44,7 @@ AC_ARG_ENABLE(threads,
 
 if test x$enable_threads = x; then
    case $host in
-    *-*-netbsd* | *-*-openbsd* )
+    *-*-netbsd* | *-*-openbsd* | *-*-mirbsd*)
      # Don't try multithreading on netbsd -- there is no threadsafe DNS
      # lookup function there.
      if test x$eventdns != xtrue; then
@@ -322,7 +322,7 @@ fi
 fi
 
 dnl Now check for particular libevent functions.
-AC_CHECK_FUNCS(event_get_version event_get_method event_set_log_callback)
+AC_CHECK_FUNCS(event_get_version event_get_method event_set_log_callback evdns_init)
 
 dnl ------------------------------------------------------
 dnl Where do you live, openssl?  And how do we call you?
@@ -492,6 +492,8 @@ dnl These headers are not essential
 
 AC_CHECK_HEADERS(stdint.h sys/types.h inttypes.h sys/param.h sys/wait.h limits.h sys/limits.h netinet/in.h arpa/inet.h machine/limits.h syslog.h sys/time.h sys/resource.h inttypes.h utime.h sys/utime.h sys/mman.h netintet/in.h netinet/in6.h sys/syslimits.h)
 
+AC_CHECK_HEADERS(sys/taitime.h)
+
 AC_CHECK_HEADERS(net/if.h, [net_if_found=1], [net_if_found=0],
 [#ifdef HAVE_SYS_TYPES_H
 #include <sys/types.h>
@@ -529,7 +531,7 @@ if test x$transparent = xtrue ; then
    if test x$transparent_ok = x1 ; then
      AC_DEFINE(USE_TRANSPARENT, 1, "Define to enable transparent proxy support")
      case $host in
-       *-*-openbsd*)
+       *-*-openbsd* | *-*-mirbsd*)
          AC_DEFINE(OPENBSD, 1, "Define to handle pf on OpenBSD properly") ;;
      esac
    else
@@ -861,9 +863,8 @@ AC_DEFINE_UNQUOTED(LOCALSTATEDIR,"$LOCAL
 # Set CFLAGS _after_ all the above checks, since our warnings are stricter
 # than autoconf's macros like.
 if test $ac_cv_c_compiler_gnu = yes; then
-  CFLAGS="$CFLAGS -Wall -g -O2"
+  CFLAGS="$CFLAGS -Wall"	# Don't set CFLAGS, MirPorts does that for us.
 else
-  CFLAGS="$CFLAGS -g -O"
   enable_gcc_warnings=no
 fi
 
